source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -53,6 +58,16 @@ a :ref:`distributed <deploy-minio-distributed>` MinIO deployment with
 Prerequisites
 -------------
 
+Add New hardware
```

Comment:
IT's a pre-req, but we lead by saying 'this is not required'.

I think we can just create the H2 where we talk about "Replacing Aged Hardware" and have the guidance there"

If anything, this section should be something like:

```
Cluster Topology Stability
~~~~~~~~~~~~~~~~~~~~~

Decommissioning requires the cluster topology remain stable throughout the process.
Do not perform expansion or similar topology alterations during the decommissioning process. 
You must complete any such alterations and verify their success before starting Decommissioning
```

or something.

Reply:
Yeah, that's a good point. This can't be a prerequisite.
Combining the suggestions into a Considerations H3 section instead.